docs/readthedocs/source/doc/LLM/Quickstart/webui_quickstart.md

```diff
@@ -40,6 +40,7 @@ pip install -r requirements_cpu_only.txt
 If you're running on iGPUs, set some environment variables by running below commands in **Anaconda Prompt**:
```

Comment:
what about dGPUs?

Reply:
Yes. But we should have

call "C:\Program Files (x86)\Intel\oneAPI\setvars.bat"

Reply:
> what about dGPUs? `"C:\Program Files (x86)\Intel\oneAPI\setvars.bat"` should be for any GPU?

Oh yes, `"C:\Program Files (x86)\Intel\oneAPI\setvars.bat"` should be in a separate command because it applies for all GPUs